### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by CaitinChen**

### CODE REVIEW

**Comment by CaitinChen** [comment]
@zz-jason I modified the English version based on Bing Duan's updates on the Chinese version. PTAL. Thanks.

File: tidb-5.0-ga-one-stop-htap-solution-deliver-speed-stability-simplicity.md [hunk 1]
```diff
@@ -48,6 +48,18 @@ Compared with version 4.0, TiDB 5.0 has achieved a noteworthy performance boost.
...
 
+Besides, TiDB 5.0 has improved its performance in OLAP scenarios by introducing the [TiFlash MPP](https://docs.pingcap.com/tidb/v5.0/use-tiflash) computing model. For a TPC-H performance test, the TiDB MPP engine's overall performance is 2 to 3 times that of Greenplum 6.15.0 or Apache Spark 3.1.1. For certain queries, the MPP engine's performance is even 7 times higher.
+
+**Hardware configuration**
+
+* For TPC-H benchmark details, see [this report](https://docs.pingcap.com/tidb/v5.0/v5.0-performance-benchmarking-with-tpch).
+
+**Workload**
+
+16 tables, each with 10 million rows of data
```
Comment (ilovesoup) [inline tidb-5.0-ga-one-stop-htap-solution-deliver-speed-stability-simplicity.md:59]: Where does this data come from? I guess its for TPC-C?

**Review by ilovesoup — APPROVED** [review]
LGTM except the comment.

**Review by ilovesoup — APPROVED** [review]
lgtm

**Review by ran-huang — APPROVED** [review]

**Pull request merged**